src/teleop/jetson/jetson_teleop.py

```diff
@@ -39,25 +39,38 @@ def __init__(self, joystick_mappings, drive_config, track_radius, wheel_radius):
         # Constants for diff drive
         self.TRACK_RADIUS = track_radius  # meter
         self.WHEEL_RADIUS = wheel_radius  # meter
-        self.drive_vel_pub = ros.Publisher("/drive_cmd_wheels", Chassis, queue_size=100)
+        self.left_wheel_pub = ros.Publisher("/drive_cmd/left_wheels", JointState, queue_size=100)
```

Comment:
ok! looks good, but i'd prefer /drive_cmd/wheels/left and /drive_cmd/wheels/right. also are we planning on making a README or nah?